staging/src/k8s.io/cloud-provider/fake/fake.go

```diff
@@ -24,7 +24,7 @@ import (
 	"sync"
 	"time"
 
-	"k8s.io/api/core/v1"
```

Comment:
I think this is left over from the last set of changes?

Reply:
ahh good catch, I thought it was formated after running ./hack/update-gofmt.sh, but actually it's not
removed!